### PR TITLE
New check: verify kms key exists and is available.

### PR DESCRIPTION
**Pull request by jcastillocano**
## Pull Request Checklist

**Is this in reference to an existing issue?**

No

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [X] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [X] Existing tests pass 

#### New Plugins

- [X] Tests

- [X] Add the plugin to the README

- [X] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose
Enable KMS key check.

#### Known Compatablity Issues
None

### CODE REVIEW

**Comment by jcastillocano** [comment]
Rubocop issues, I ran it with ruby 2.2.1, reviewing with all version (2.0.0, 2.1.0, 2.2.0)

File: bin/check-kms-key.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class CheckKMSKey < Sensu::Plugin::Check::CLI
+  option :aws_access_key,
+         short:       '-a AWS_ACCESS_KEY',
```
Comment (sstarcher) [inline bin/check-kms-key.rb]: Please remove

File: bin/check-kms-key.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+         long:        '--aws-access-key AWS_ACCESS_KEY',
+         description: "AWS Access Key. Either set ENV['AWS_ACCESS_KEY'] or provide it as an option",
+         default:     ENV['AWS_ACCESS_KEY']
+
+  option :aws_secret_access_key,
```
Comment (sstarcher) [inline bin/check-kms-key.rb]: Please remove

File: bin/check-kms-key.rb [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+#   Copyright 2016 github.com/jcastillocano
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'aws-sdk'
+
+class CheckKMSKey < Sensu::Plugin::Check::CLI
+  option :aws_access_key,
```
Comment (sstarcher) [inline bin/check-kms-key.rb]: import the Common library

File: bin/check-kms-key.rb [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+         description: 'AWS Region (defaults to us-east-1).',
+         default:     'us-east-1'
+
+  option :key_id,
+         short:       '-k ID',
+         long:        '--key-id ID',
+         description: 'KMS key identifier',
+         default:     nil
+
+  def aws_config
```
Comment (sstarcher) [inline bin/check-kms-key.rb]: remove

File: bin/check-kms-key.rb [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    { access_key_id: config[:aws_access_key],
+      secret_access_key: config[:aws_secret_access_key],
+      region: config[:aws_region]
+    }
+  end
+
+  def aws_client(opts = {})
```
Comment (sstarcher) [inline bin/check-kms-key.rb]: remove

**Comment by jcastillocano** [comment]
Using Common library.

**Pull request merged**